.gitignore

```diff
@@ -18,6 +18,8 @@ sim/web/__debug_bin
 
 # temporary files
 *.results.tmp
+temp.json
```

Comment:
What do you think about naming something like `package-tmp.json?` That was it's more clear that this is a copy of a `package.json` file?

Reply:
> What do you think about naming something like `package-tmp.json?` That was it's more clear that this is a copy of a `package.json` file?

new day, new me. 

`package.json.tmp` okay?

Reply:
@dexxiez Yea that sounds fine. Also that file shouldn't exist after modifying anyways, right?
As in, it uses the `mv` command to make the temp, but then swap it back and run `npm install`

Reply:
Yeah under normal circumstances it will turn into a normal `package.json`. I'm no low level expert but I think the main reasoning behind the temp file write is to do with stream safety to not directly write back to the original file instantly. 

This line in the ignore file just accounts for glitches in space, time and quantum mechanics etc

Reply:
Yea was just double checking I didn't miss anything.